pages/ActivityStreams/Overview.md

```diff
@@ -33,22 +33,347 @@ These deviations from the standard will be thoroughly documented here.
+If these fields are not provided, clients may attempt to infer the media type from the headers of the linked file or simply ignore the object.
+Clients may also ignore video files of exceptionally large or small sizes, however videos with a preview of one of the prior mentioned resolutions must always be supported.
+
+For example, the following would be valid video objects:
+
+```json
+{
+  "@context": "https://www.w3.org/ns/activitystreams",
+  "type": "Video",
+  "name": "Puppy Plays With Ball",
+  "url": "http://example.org/video.mkv",
+  "duration": "PT2H"
+}
+```
+
+```json
+{
+  "@context": "https://www.w3.org/ns/activitystreams",
+  "type": "Video",
+  "name": "Big Buck Bunny",
+  "url": {
+    "type": "Link",
+    "href": "https://upload.wikimedia.org/wikipedia/commons/c/c0/Big_Buck_Bunny_4K.webm"
+  },
+  "duration": "PT10M32S",
+  "mediaType": "video/webm",
+  "width": 4000,
+  "height": 2250
+}
+```
+
 ## Extensions
 
 In addition to the standards defined by the Activity Streams specifications, some extensions of Activity Streams are also required by the DSNP due to its decentralized context.
 
-### Required Keys and Values
+### Content Contexts
+
+Clients may sometimes want to represent content as existing within a specific presentation or social context, such as Reddit's subreddit concept, Slack's channels or old-style forum rooms.
+It is recommended that implementers represent this context in activity objects with the `"context"` field.
+If present, this field must have a string value starting with the name of the implementing service followed by forward slash and a string unique to the specific context.
+Implementers are encouraged to add additional forward slash separators to their unique string to represent contexts within contexts.
+For example, a `#development` channel context within a `LibertyDSNP` organization on Slack could be represented by the string, `"Slack/LibertyDSNP/#development"`.
+
+Clients may choose to ignore activity objects with contexts outside the user's current context, but it is strongly recommended that all contexts be made accessible to the user through some form of navigation.
+
+### Deletion Support
+
+Users may occasionally want to delete an activity object by changing it's contents, thereby changing it's hash value and invalidating any signature associated with it via broadcast or reply messages on chain.
+Likewise, hosts of activity objects may want to do the same for the purpose of removing illegal or otherwise objectionable content.
+Implementers are strongly recommended to represent this by adding a `"deleted"` field to the hosted activity object with an [ISO8601](https://www.iso.org/iso-8601-date-and-time-format.html) timestamp representing the time at which the content was removed.
```

Comment:
What does strongly recommended mean? Can we just say "must"? Why would implementers not be able to do this?

Reply:
My thinking was that we can't require this because there's no way to prevent users from invalidating their activity pubs without adding a deleted timestamp.

I'll change it to "should."